backend/src/config/db.ts

```diff
@@ -0,0 +1,16 @@
+import pg from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { Pool } = pg;
+
+console.log(process.env.DB_PASSWORD);
+
+export const pool = new Pool({
+  user: process.env.DB_USERNAME,
+  database: process.env.DB_NAME,
+  password: process.env.DB_PASSWORD,
+  port: Number(process.env.DB_PORT),
+  host: process.env.DB_HOST
+});
```

Comment:
I think a good practice is to put a new line at the end of the file.

Reply:
Good idea. I'll add add a linter to the todo list.